app/org/sagebionetworks/bridge/services/backfill/AsyncBackfillTemplate.java

```diff
@@ -30,15 +30,20 @@
     private BackfillRecordFactory backfillRecordFactory;
 
     @Autowired
-    public void setDistributedLockDao(DistributedLockDao lockDao) {
+    public final void setDistributedLockDao(DistributedLockDao lockDao) {
         this.lockDao = lockDao;
     }
     @Autowired
-    public void setBackfillDao(BackfillDao backfillDao) {
+    public final void setBackfillDao(BackfillDao backfillDao) {
         this.backfillDao = backfillDao;
     }
+
+    public BackfillRecordFactory getBackfillRecordFactory() {
+        return backfillRecordFactory;
+    }
+
     @Autowired
-    public void setBackfillRecordFactory(BackfillRecordFactory backfillRecordFactory) {
+    public final void setBackfillRecordFactory(BackfillRecordFactory backfillRecordFactory) {
```

Comment:
The use of "final" would be a nice thing to add to the standards document. We could add final all over the place.